packages/core/src/Stores/common-store.js

```diff
@@ -259,6 +259,13 @@ export default class CommonStore extends BaseStore {
         });
     };
 
+    @action.bound
+    getExchangeRate = async (from_currency, to_currency) => {
+        const { exchange_rates } = await BinarySocket.exchange_rates(from_currency);
+
+        return exchange_rates?.rates?.[to_currency];
+    };
+
```

Comment:
You can import it from the `core/src/Utils`, no need to redefine.

Reply:
since we are using it in the appstore and we do not have access to core utils, it should be kept in core stores